internal/provider/models/destinations/azure_blob_storage.go

```diff
@@ -28,6 +28,8 @@ type AzureBlobStorageDestinationModel struct {
 	ContainerName       String `tfsdk:"container_name" user_config:"true"`
 	ConnectionString    String `tfsdk:"connection_string" user_config:"true"`
 	Prefix              String `tfsdk:"prefix" user_config:"true"`
+
```

Comment:
Remove blank line

Reply:
I put that here intentionally to make the diff easier to read. Gofmt will make whitespace changes to every other field in the struct. But I can change it.